portfolio/src/main/webapp/index.html

```diff
@@ -12,30 +12,42 @@
-                    <li class="card-panel transparent tab row s1 center-align"><a href="#projetcs" class="grey-text text-lighten-3 nav-icon" style="font-size: 1vw">Projects</a></li>
-                    <li class="card-panel transparent tab row s1 center-align"><a href="#activities" class="grey-text text-lighten-3 nav-icon" style="font-size: 1vw">Activities</a></li>
-                    <li class="card-panel transparent tab row s1 center-align"><a href="#travels" class="grey-text text-lighten-3 nav-icon" style="font-size: 1vw">Travels</a></li>
-                    <li class="card-panel transparent tab row s1 center-align"><a href="#hobbies" class="grey-text text-lighten-3 nav-icon" style="font-size: 1vw">Hobbies</a></li>
+        <nav class="row" style="background-color: gray;">
+            <div class="nav-wrapper">
+                <ul id="nav-mobile" class="right">
+                    <li><a id="log-in-out" href="/login" class="grey-text text-lighten-3 nav-icon" style="font-size: 1.5vw"
+                         onclick="fetch('/login')">Login/Logout</a></li>
+                </ul>
+                <ul id="nav-mobile" class="lfet">       
+                    <li><a href="#about-me" class="grey-text text-lighten-3 nav-icon" style="font-size: 1.5vw">About Me</a></li>
+                    <li><a href="#experience" class="grey-text text-lighten-3 nav-icon" style="font-size: 1.5vw">Experience</a></li>
+                    <li><a href="#projetcs" class="grey-text text-lighten-3 nav-icon" style="font-size: 1.5vw">Projects</a></li>
+                    <li><a href="#activities" class="grey-text text-lighten-3 nav-icon" style="font-size: 1.5vw">Activities</a></li>
+                    <li><a href="#travels" class="grey-text text-lighten-3 nav-icon" style="font-size: 1.5vw">Travels</a></li>
+                    <li><a href="#hobbies" class="grey-text text-lighten-3 nav-icon" style="font-size: 1.5vw">Hobbies</a></li>
                 </ul>
             </div>
-            <div class="col s1"></div>
+        </nav>
+        <div class="row my-wrapper valign-wrapper">
+            <div class="col s2 center-align">
+                <div id="chat-room" class="card-panel center-align">
+                    <div class="rwo"><h5 class="grey-text text-lighten-3 center-align" style="font-size: 1.4vw;">Questions & Answers</h5></div>
+                    <div id="questions-answers" class="row left-align"></div>
+                    <div id="load-more-container" class="center-align row"></div>
+                </div>
+            </div>
             <div class="col s8 center-align">
                 <div class="row header">
                     <div class="divider about-me-divider"></div> 
-                    <h1 class="center-align intro-text" style="font-size:4vw; margin: 0">NABIL DABOUZ</h1>
+                    <h1 class="center-align intro-text" style="font-size:3vw; margin: 0">NABIL DABOUZ</h1>
```

Comment:
It seems that you are using this style for all your h1 headers. Include it in your css?